packages/mask/src/plugins/IdeaMarket/hooks/useFetchUserTokens.ts

```diff
@@ -0,0 +1,24 @@
+import { useAsync } from 'react-use'
+import { fetchTwitterLookup, fetchUserTokensBalances } from '../apis'
+import type { UserIdeaTokenBalance } from '../types'
+
+export function useFetchUserTokens(holder: string) {
+    return useAsync(async () => {
+        if (!holder) return
+        const tokenBalances = await fetchUserTokensBalances(holder)
+
+        const tokenBalancesWithTwitterData = await Promise.all(
+            tokenBalances.map(async (balance: UserIdeaTokenBalance) => {
+                if (balance.token.market.id !== '0x1') return balance
```

Comment:
Please compare it with `ChainId`.

Reply:
`market` is associated to a social network not a chain, so I modified it accordingly.